draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -564,7 +600,12 @@ in (#hash_function_claim)).
 
 ### Validating the Key Binding JWT
 
-To validate the signature on the Key Binding JWT, the Verifier MUST use the key material in the SD-JWT. If it is not clear from the SD-JWT, the Key Binding JWT MUST specify which key material the Verifier needs to use to validate the Key Binding JWT signature using JOSE header parameters such as `kid` and `x5c`.
+The Verifier MUST ensure that the key with which it validates the signature on
+the Key Binding JWT is the key specified in the SD-JWT as the using the Holder's
+public key.  For example, if the SD-JWT contains a `cnf` value of type `jkt`,
+the Verifier would confirm that the public key's JWK Thumbprint matches the `jkt`
+value.  Or if the SD-JWT contains a `cnf` value of type `jwk`, then the Verifier
```

Comment:
what is "`cnf` value of type `jkt`" (i don't think it is defined in rfc7800?) and why is it used as a main example?

Reply:
it lets you use a thumbprint to resolve a public key, instead of sending the public key by value. there is an analog in COSE.

Reply:
It was defined (and registered) by DPoP https://www.rfc-editor.org/rfc/rfc9449.html#section-6.1 FWIW. But to @Sakurann's implied point, it's probably not appropriate as the main example in this draft. Or necessarily appropriate/needed as an example at all in this draft. I think the `jwk` confirmation method should be encouraged as much as possible in the SD-JWT context.